libs/designsystem/card/src/card-as-button/card-as-button.directive.spec.ts

```diff
@@ -1,15 +1,23 @@
 import { createDirectiveFactory, SpectatorDirective } from '@ngneat/spectator';
+import { Component } from '@angular/core';
 import { CardComponent } from '../card.component';
 import { CardAsButtonDirective } from './card-as-button.directive';
 
 describe('CardAsButtonDirective', () => {
   let spectator: SpectatorDirective<CardAsButtonDirective>;
   let cardElement: HTMLElement;
 
+  @Component({ selector: 'kirby-card-click-host', template: '' })
+  class KirbyCardClickHostComponent {
+    someMethod() {
+      //noop
+    }
+  }
```

Comment:
I think the component should be moved to line 5 for readability, so it is not nested within the describe block.